hapistrano.cabal

```diff
@@ -28,7 +28,7 @@ category:            System
 homepage:            https://github.com/stackbuilders/hapistrano
 bug-reports:         https://github.com/stackbuilders/hapistrano/issues
 build-type:          Simple
-tested-with:         GHC==8.10.17, GHC==9.0.2
+tested-with:         GHC==9.6.5
```

Comment:
```suggestion
tested-with:         GHC==9.6.6
```

Reply:
@DavidMazarro I believe we should also bump the version of `base` at some point; however, I believe it would be best to do so in a separate PR because it would most likely require a major release.